PowerEditor/installer/themes/Zenburn.xml

```diff
@@ -535,6 +535,24 @@ License:             GPL2
             <WordsStyle name="COMMENT BLOCK2" styleID="15" fgColor="7F9F7F" bgColor="3F3F3F" fontName="" fontStyle="2" fontSize="" />
             <WordsStyle name="COMMENT BLOCK3" styleID="16" fgColor="7F9F7F" bgColor="3F3F3F" fontName="" fontStyle="2" fontSize="" />
         </LexerType>
+        </LexerType>

```

Comment:
The repeated `</LexerType>` end-tag is [failing the XML check](https://ci.appveyor.com/project/donho/notepad-plus-plus/builds/46605476/job/ycs4q01ex82lncck/messages).

Try this:

~~~diff
diff --git a/PowerEditor/installer/themes/Zenburn.xml b/PowerEditor/installer/themes/Zenburn.xml
index 800e3341..b8025e09 100644
--- a/PowerEditor/installer/themes/Zenburn.xml
+++ b/PowerEditor/installer/themes/Zenburn.xml
@@ -535,7 +535,6 @@ License:             GPL2
             <WordsStyle name="COMMENT BLOCK2" styleID="15" fgColor="7F9F7F" bgColor="3F3F3F" fontName="" fontStyle="2" fontSize="" />
             <WordsStyle name="COMMENT BLOCK3" styleID="16" fgColor="7F9F7F" bgColor="3F3F3F" fontName="" fontStyle="2" fontSize="" />
         </LexerType>
-        </LexerType>
         <LexerType name="hollywood" desc="Hollywood" ext="">
             <WordsStyle name="DEFAULT" styleID="0" fgColor="DCDCCC" bgColor="3F3F3F" fontName="" fontStyle="0" fontSize="" />
             <WordsStyle name="COMMENT" styleID="1" fgColor="7F9F7F" bgColor="3F3F3F" fontName="" fontStyle="0" fontSize="" />

~~~

Reply:
@afalkenhahn 
Please consider @rdipardo's suggestion to remove close tag `</LexerType>`